### PR TITLE
chore: Packaging maintenance

### PR DESCRIPTION
**Pull request by crccheck**
Basic maintenance and update some package.json cobwebs.

* This uses a known license, it should use the right syntax: https://docs.npmjs.com/files/package.json#license
* The npm scripts used to run eslint multiple times
* Trying to reduce the friction to contribute

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,13 @@
-<!--- Provide a general summary of your changes in the Title above -->
-
-## Description
+<!--- Summarize your changes in the title above -->
```
Comment (pgoldrbx) [inline .github/PULL_REQUEST_TEMPLATE.md]: This template was part of our shared boilerplate that we agreed to use for OSS projects
https://github.com/CondeNast/opensource-check/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: package.json [hunk 2]
```diff
@@ -20,26 +27,20 @@
...
-    "index.min.js"
-  ],
-  "dependencies": {},
-  "browserslist": [
-    "last 2 years"
-  ]
+    "mocha": "^5.2.0",
+    "sinon": "^7.2.2",
+    "uglify-js": "^3.4.9"
+  }
```
Comment (pgoldrbx) [inline package.json]: can we add greenkeeper or renovate?

**Comment by crccheck** [comment]
I was doing a scan of our public face of npm https://npmjs.com/~condenast and quick-bus stuck out to me as something that could use a revisit. I'm not sure if we need Renovate here because we don't do a lot here so the Renovate commits will add a lot of noise to the commit history.

Heck, I'm not even sure if this PR is worth it, but I felt like updating the `"LICENSE in LICENSE"` so tools that use `package.json` get a more accurate picture of this project.

**Comment by mankins** [comment]
This looks good, any reason not to merge this?

**Comment by crccheck** [comment]
I think it was a combination of me forgetting about it over the Christmas break, and me concerned about doing maintenance while not actually using this project so I'm not in the feedback loop for the impact of this PR.

**Comment by pgoldrbx** [comment]
Looks like main consumers are ads and verso?
https://github.com/search?l=JSON&q=org%3ACondeNast+quick-bus&type=Code

**Comment by mankins** [comment]
@pgoldrbx that's right. The repo is public but I couldn't find others using it.

@crccheck we could merge this and my outstanding pr, with a version bump?

**Comment by crccheck** [comment]
ok. Going to merge this. It's a really safe change as it doesn't touch the code and just reorders the package.json

**Comment by crccheck** [comment]
oh wait... I don't have permission to merge in here

**Comment by natelaws** [comment]
> oh wait... I don't have permission to merge in here

Just updated the team permissions...

File: package.json (hunk 2 above)
Comment (landau) [reply package.json]: and `semantic-release` :party-parrot:

**Pull request merged**